### ISSUE TITLE
SlevomatCodingStandard.PHP.UselessParentheses.UselessParentheses false positive

### ISSUE THREAD

**Issue by mforbak** [slevomat/coding-standard#1202]
When using something like this `$char = (0xF0 & ord('1')) >> 4;` phpcs returns me:
**Useless parentheses** (SlevomatCodingStandard.PHP.UselessParentheses.UselessParentheses)`
```php
$char = (0xF0 & ord('1')) >> 4;
var_dump($char); // int(3)

$char = 0xF0 & ord('1') >> 4;
var_dump($char); // int(0)
```
But after removing parentheses, the meaning is absolutely different.

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/1fa3034e769a4dc1d2cb10ee2e675efdec341075
